client/my-sites/customer-home/main.jsx

```diff
@@ -49,6 +50,14 @@ const Home = ( {
 	trackViewSiteAction,
 } ) => {
 	const translate = useTranslate();
+	// use an effect to fire the exposure event only if the site id changes or first view
```

Comment:
Maybe leave a comment here that we're temporarily testing the experimentation framework as an A/A test and that we expect to remove this in +3 weeks or so.

Reply:
I moved this to the experiment component, so as to not bloat this component (and future experiments)